### PR TITLE
Simplify Broadway by removing pod manifests

### PR DESCRIPTION
**Pull request by macat**
- remove tasks and pod manifests from playbooks

### CODE REVIEW

**Comment by poopoothegorilla** [comment]
+1

File: pkg/deployment/playbook_test.go [hunk 1]
```diff
@@ -25,44 +25,22 @@ vars:
...
+name: The Project
 messages:
   created: "help I'm stuck in a test {{ .ID }}"
   deployed: "help I'm still stuck in a test {{ .ID"
-tasks:
-  - name: Deploy Postgres
-    manifests:
-      - hello
+manifests:
+  - hellooo
```
Comment (therealplato) [inline pkg/deployment/playbook_test.go]: let's use `hello` here so there's no ambiguity about whether the failure is the bad template or the missing manifest

File: pkg/server/server_test.go [hunk 2]
```diff
@@ -22,13 +22,15 @@ import (
...
 var testCfg = cfg.Type{
+	K8sNamespace:       "broadway",
 	AuthBearerToken:    "testtoken",
 	SlackToken:         testToken,
 	ManifestsPath:      "../../examples/manifests",
 	ManifestsExtension: ".yml",
 	PlaybooksPath:      "../../examples/playbooks",
 	EtcdEndpoints:      "http://localhost:4001",
 	EtcdPath:           "/broadwaytest",
+	InstanceCleanup:    15,
```
Comment (therealplato) [inline pkg/server/server_test.go]: can these vars be removed from this PR?

**Comment by therealplato** [comment]
+1 but see comments @macat

File: pkg/server/server_test.go (hunk 2 above)
Comment (macat) [inline pkg/server/server_test.go]: No, actually these are being used.

**Pull request merged**